core/src/main/java/org/eclipse/hono/util/MessageHelper.java

```diff
@@ -529,4 +563,59 @@ public static void setCreationTime(final Message msg) {
         }
     }
 
+    /**
+     * Verify if a message is signalling that the
+     * device for which it was received should be ready to receive a command.
+     * This is evaluated at the point in time this method is invoked.
+     *
+     * @param msg Message that is evaluated.
+     * @return Boolean {@code true} if the message signals that the device now should be ready to receive a command, {@code false}
+     * otherwise.
+     * @throws NullPointerException If msg is {@code null}.
+     */
+    public static Boolean isDeviceCurrentlyReadyForCommands(final Message msg) {
```

Comment:
this method checks if a device is (supposedly) currently connected. Whether this is used in order to send a command or not has nothing to do with this method, does it?

IMHO it should be renamed and the JavaDoc should be updated accordingly.

Reply:
I agree, I will rename it :+1: